src/main/java/learnositysdk/request/Telemetry.java

```diff
@@ -0,0 +1,160 @@
+package learnositysdk.request;
+
+import org.apache.commons.io.IOUtils;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.nio.charset.Charset;
+import java.util.Locale;
+import java.util.Properties;
+
+class Telemetry {
+	private static boolean telemetryEnabled = true;
+	private static JSONObject telemetryData = null;
+
+	// Java Runtimes (as expected in lib-metrics)
+	private final static String RT_OPEN_JDK = "openjdk";
```

Comment:
Can we not get those from the runtime themselves?

Reply:
Oracle Java Runtime: `java.runtime.name = Java(TM) SE Runtime Environment`
OpenJDK Runtime: `java.runtime.name = OpenJDK Runtime Environment`

I wasn't feeling particularly keen on sending the whole string since we end up just filtering it down to those two values anyway:
https://github.com/Learnosity/lib-metrics-php/blob/master/src/SdkMetricsParser.php#L11

Reply:
Yeah, fair enough. I would still prefer to match on, say,  `Java(TM)`, and report unknown for anything else.